src/scripts/buildConfigs/routes.ts

```diff
@@ -1,15 +1,37 @@
-import { ProtocolType, bytesToProtocolAddress, eqAddress } from '@hyperlane-xyz/utils';
+import { ProtocolType, bytesToProtocolAddress, deepCopy, eqAddress } from '@hyperlane-xyz/utils';
+
+import { ibcRoutes } from '../../consts/ibcRoutes';
+import { WarpContext } from '../../context/context';
+import { getCaip2Id } from '../../features/caip/chains';
+import { getChainIdFromToken, isNonFungibleToken } from '../../features/caip/tokens';
+import { Route, RouteType, RoutesMap } from '../../features/routes/types';
+import { AdapterFactory } from '../../features/tokens/AdapterFactory';
+import { isIbcToken } from '../../features/tokens/metadata';
+import { TokenMetadata, TokenMetadataWithHypTokens } from '../../features/tokens/types';
+import { logger } from '../../utils/logger';
+
+export async function getRouteConfigs(context: WarpContext): Promise<RoutesMap> {
+  logger.info('Searching for token routes');
+  const processedTokens: TokenMetadataWithHypTokens[] = [];
+  for (const token of context.tokens) {
+    // Skip querying of IBC tokens
+    if (isIbcToken(token)) continue;
+    const tokenWithHypTokens = await fetchRemoteHypTokens(context, token, context.tokens);
```

Comment:
can remove the 3rd arg from `fetchRemoteHypTokens` if we're already passing through the full context?

Reply:
Good catch, thanks